src/jquery.uls.lcd.js

```diff
@@ -416,14 +423,23 @@
 	};
 
 	$.fn.lcd.defaults = {
-		languages: null,
+		// List of languages to show
+		languages: [],
+		// List of regions to show
 		showRegions: [ 'WW', 'AM', 'EU', 'ME', 'AF', 'AS', 'PA' ],
+		// Whether to group by region, defaults to true when columns > 1
+		groupByRegion: undefined,
```

Comment:
Can it be false? if undefined has a different meaning other than false, I think we should clearly document to avoid future confusion. The documentaiton in README.md is also unclear about what is undefined

Reply:
It cannot be false, because the default value is determined during runtime if not set.